tests/test_dynamic_resource.py

```diff
@@ -130,14 +128,6 @@ def _render(state):
 <h1>Greetings, program!</h1>
 """
 
-def test_render_renders(harness):
```

Comment:
It was only testing that `output` could be introduced into the state early, which is no longer the case.